.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature Request
+about: Proposal for a new Chalk Feature
+---
+
+## Description
+
+<!-- what new feature would you like to see in chalk? please include a link to the associated high-level objective -->
+
+## Result
+
+<!-- brief expectations of how end result should look + behave -->
+
+## Dependencies
+
+<!-- are there any tickets/prs that should be worked on before this feature can be added? -->
+
+## Subtickets
+
+<!-- if it makes sense to break this ticket up into smaller pieces of work, link them here -->
```

Comment:
For the link back to objectives, could just broadly make this "Related Tickets"?